src/hashtables.jl

```diff
@@ -123,6 +126,8 @@ set_maxOrder(n::Int) = set_params_TaylorN(n, _params_taylorN.numVars)
 get_numVars() = _params_taylorN.numVars
 set_numVars(n::Int) = set_params_TaylorN(_params_taylorN.maxOrder, n)
 
+set_variable_names(names::Vector{UTF8String}) = _params_taylorN.variable_names = names
+
 function set_params_TaylorN(order::Int, numVars::Int)
```

Comment:
Below (line 138) needs changes, since `ParamsTaylorN` now has three field names; I think tests are not passing because of this line. A way around is to simply add another method for `ParamsTaylorN`